packages/js/components/src/plugins/index.tsx

```diff
@@ -0,0 +1,156 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import {
+	createElement,
+	Fragment,
+	useState,
+	useEffect,
+} from '@wordpress/element';
+import { SyntheticEvent } from 'react';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { PLUGINS_STORE_NAME, InstallPluginsResponse } from '@woocommerce/data';
+
+type PluginsProps = {
+	onComplete: (
+		activePlugins: string[],
+		response: InstallPluginsResponse
+	) => void;
+	onError: ( errors: unknown, response: InstallPluginsResponse ) => void;
```

Comment:
Changed this from `onError` because you declare a default value it is never null.